swift/Tests/CosmosSignerTests.swift

```diff
@@ -17,29 +17,60 @@ class CosmosSignerTests: XCTestCase {
         var sendCoinsMessage = TW_Cosmos_Proto_SendCoinsMessage()
         sendCoinsMessage.fromAddress = fromAddress
         sendCoinsMessage.toAddress = "cosmos1zt50azupanqlfam5afhv3hexwyutnukeh4c573"
-        var coin = TW_Cosmos_Proto_Coin()
-        coin.amount = 1
-        coin.denom = "muon"
-        sendCoinsMessage.amount = [coin]
+        var txAmount = TW_Cosmos_Proto_Amount()
```

Comment:
checkout `Cosmos+Proto.swift`, we generate swift typealias automatically, so you can use `CosmosAmount`

Reply:
done in 3a15f10707237aa97320142331a0a3171c3e8cdb